TransferMethod/Sources/AddTransferMethodController.swift

```diff
@@ -48,6 +48,8 @@ final class AddTransferMethodController: UITableViewController {
     private var presenter: AddTransferMethodPresenter!
     private var widgets = [AbstractWidget]()
     private var isCalledByScrollToRow = false
+    private let pageName = "transfer-method:add:collect-transfer-method-information"
+    private let pageGroup = "transfer-method"
```

Comment:
Why do you need them here? These are already defined in Presenter

Reply:
Removed -  This was added before showError method in presenter  was modified to pass pagename and pagegroup to controller